test/fixtures/petstore.graphql

```diff
@@ -151,14 +94,14 @@ type Query {
   getPetById(petId: String!): Pet!
 
   """Returns a map of status codes to quantities"""
-  getInventory: getInventory!
+  getInventory: getInventory_response!
 
   """
   For valid response try integer IDs with value >= 1 and <= 10. Other values will generated exceptions
   """
   getOrderById(orderId: String!): Order!
-  loginUser(username: String!, password: String!): loginUser!
-  logoutUser: logoutUser!
+  loginUser(username: String!, password: String!): String!
```

Comment:
Although the graphql types are much better, it still won't return the right result because swagger-to-graphql tries to parse the string as json

Reply:
fixed!